### PR TITLE
Favicon fix

### PR DESCRIPTION
**Pull request by s-varun**

### CODE REVIEW

**Comment by AliceYuan** [comment]
( merge your commits)

**Comment by MeenuV** [comment]
squash* ?

Thanks,
Meenu

**Comment by MeenuV** [comment]
build is failing...

File: test/client/scraper/test_general_metadata.py [hunk 1]
```diff
@@ -28,6 +29,23 @@ def test_general(self):
             self.assertTrue(json_response['data']['images']['count'] > 0)
             self.assertEqual(json_response['status'], 200)
 
+    def test_general2(self):
```
Comment (MeenuV) [inline test/client/scraper/test_general_metadata.py]: Can we make a better name for this test?

Comment (AliceYuan) [inline test/client/scraper/test_general_metadata.py]: this seems a little redundant, can we instead do a for loop with the different test urls

Comment (AliceYuan) [inline test/client/scraper/test_general_metadata.py]: also add in a couple more urls? try the yelp & apple one

**Comment by AliceYuan** [comment]
r+

**Pull request merged**